src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -201,26 +201,35 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
               </div>
 
               <div className="input-group">
-                <label htmlFor="date">Starting time</label>
-                <label>
-                  When does this event start? Use your local time zone, it will
-                  be automatically converted for anyone visiting from around the
-                  world.
+                <label htmlFor="date">
+                  Start date and time (use your own time zone; it will be
+                  automatically localized)
                 </label>
+                <label></label>
```

Comment:
[NOTE]
This is just legacy way of "doing business". The next PR can re-use some of `SpaceTimingForm` components: 

```tsx
          <AdminDateTime
            name="end"
            register={register}
            errors={errors}
            label="Global ending time"
          />
```

There are few more props in https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/AdminDateTime/AdminDateTime.tsx#L16

Reply:
https://github.com/sparkletown/sparkle/pull/2599/commits/7eb452fae9982ead81affd8ed7ad8664c6fb813e